internal/api/grpc/user/v2/user_test.go

```diff
@@ -407,3 +407,7 @@ func Test_authMethodTypeToPb(t *testing.T) {
 		})
 	}
 }
+
+func ptr[T any](v T) *T {
```

Comment:
We already have a helper function for this: `gu.Ptr`. Please remove this and use that in all occurences.